src/plugin/cesium/cesiumcamera.js

```diff
@@ -377,545 +426,528 @@ plugin.cesium.Camera.prototype.initExtentComputation_ = function() {
-    altitude = Math.min(altitude, maxAltitude);
-
-    var heading = options.heading != null ? ol.math.toRadians(options.heading) : this.cam_.heading;
-    // KML standard pitch is 0 to look directly at the Earth, Cesium is -90
-    var pitch = options.pitch != null ? ol.math.toRadians(options.pitch - 90) : this.cam_.pitch;
-    var roll = options.roll != null ? ol.math.toRadians(options.roll) : this.cam_.roll;
-
-    if (options.positionCamera) {
-      // move the camera to the specified position
-      target.height = altitude;
-
-      this.cam_.flyTo({
-        destination: Cesium.Ellipsoid.WGS84.cartographicToCartesian(target),
-        duration: duration / 1000,
-        easingFunction: easingFunction,
-        orientation: {
-          heading: heading,
-          pitch: pitch,
-          roll: roll
-        }
-      });
-    } else {
-      // point the camera at the specified position, on the ground
-      target.height = 0;
-
-      var boundingSphere = new Cesium.BoundingSphere(Cesium.Ellipsoid.WGS84.cartographicToCartesian(target));
-      var range = options.range != null ? options.range : altitude;
-      var offset = new Cesium.HeadingPitchRange(heading, pitch, range);
-
-      this.cam_.flyToBoundingSphere(boundingSphere, {
-        offset: offset,
-        duration: duration / 1000,
-        easingFunction: easingFunction
-      });
+  /**
+   * @inheritDoc
+   */
+  rotateDown(opt_value) {
+    if (this.cam_) {
+      this.cam_.rotateDown(opt_value);
     }
   }
-};
 
+  /**
+   * @inheritDoc
+   */
+  twistLeft(opt_value) {
+    if (this.cam_) {
+      this.cam_.twistLeft(opt_value);
+    }
+  }
 
-/**
- * @inheritDoc
- */
-plugin.cesium.Camera.prototype.getDistanceToCenter = function() {
-  var center;
-  var canvas = /** @type {HTMLCanvasElement} */ (document.querySelector(os.map.WEBGL_CANVAS));
-  if (canvas) {
-    center = this.cam_.pickEllipsoid(new Cesium.Cartesian2(canvas.width / 2, canvas.height / 2));
+  /**
+   * @inheritDoc
+   */
+  twistRight(opt_value) {
+    if (this.cam_) {
+      this.cam_.twistRight(opt_value);
+    }
   }
 
-  // try to base zoom on the distance from the camera to the globe at the center of the screen, otherwise use the
-  // camera's current altitude from the globe.
-  return center ? Cesium.Cartesian3.distance(this.cam_.positionWC, center) : this.getAltitude();
-};
+  /**
+   * @inheritDoc
+   */
+  cancelFlight() {
+    if (this.enabled_ && this.cam_) {
+      this.cam_.cancelFlight();
+    }
+  }
 
+  /**
+   * @inheritDoc
+   */
+  flyTo(options) {
+    if (this.enabled_) {
+      var duration = options.duration != null ? options.duration : MapContainer.FLY_ZOOM_DURATION;
+      var easingFunction = options.flightMode === FlightMode.SMOOTH ?
+        Cesium.EasingFunction.LINEAR_NONE : undefined;
+
+      var target;
+      if (options.center) {
+        var center = proj.transform(options.center, osMap.PROJECTION, os.proj.EPSG4326);
```

Comment:
Another global reference for `os.proj`. Given how long this PR is, there are probably a few of these left. Not sure if we want to bother picking through it for them or just dealing with them later.

Reply:
Yeah we can write a transform to do these later.